_data/Bhavana22-cmd/Bhavana22-cmd.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Bhavana Goud 
+institution: CVR College Of Engineering 🚩
```

Comment:
Hi @Bhavana22-cmd, it seems like there is a "flag icon" at the end of the institution. I think it would be best if you removed this to not cause any issues with merging the PR.

This would help moderators review your PR correctly.

Reply:
Thank you , I have removed it.